kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx

```diff
@@ -1,67 +1,164 @@
 import React from 'react';
-import { StaticRouter } from 'react-router';
+import { screen, within } from '@testing-library/react';
+import { render } from 'lib/testHelpers';
 import TableHeaderCell, {
   TableHeaderCellProps,
 } from 'components/common/table/TableHeaderCell/TableHeaderCell';
-import { mountWithTheme } from 'lib/testHelpers';
 import { TopicColumnsToSort } from 'generated-sources';
+import theme from 'theme/theme';
+import userEvent from '@testing-library/user-event';
 
-const STUB_TITLE = 'stub test title';
-const STUB_PREVIEW_TEXT = 'stub preview text';
+const title = 'test title';
+const previewText = 'test preview text';
+const orderBy = TopicColumnsToSort.NAME;
+const orderValue = TopicColumnsToSort.NAME;
+const otherOrderValue = TopicColumnsToSort.OUT_OF_SYNC_REPLICAS;
```

Comment:
I tried to change them like this and I don't like how it looks
If we, by some reason, need to change `const orderByName = TopicColumnsToSort.NAME` to something like `const orderByTotalPartitions = TopicColumnsToSort.TOTAL_PARTITIONS` we have to change all variables to new one

I'm thinking about removing `orderBy`, `orderValue` and `otherOrderValue` variables and to use `TopicColumnsToSort.NAME` etc 
It will be more readable, I think

Reply:
![image](https://user-images.githubusercontent.com/12596177/149927070-a54c7393-6aca-42db-9666-478bbe4c1f2e.png)
Looks better IMO

Reply:
Yes it is also a good version.